master/buildbot/test/unit/test_reporter_http.py

```diff
@@ -77,7 +77,7 @@ def test_basic(self):
         yield self.createReporter()
         build = yield self.setupBuildResults(SUCCESS)
         build['complete'] = False
-        self.sp.buildStarted(("build", 20, "started"), build)
+        self.sp.buildStarted(("build", 20, "new"), build)
```

Comment:
this probably should be added somewhere more explicitly (this "preference" follows PEP-8)